tutorial/developing_packages/editable_packages.rst

```diff
@@ -0,0 +1,204 @@
+For those cases, you can put your package in editable mode, and consumers will be able to
+find the headers and artifacts in your local working directory, eliminating the need for
+packaging.
+
+Let's see how we can put a package in editable mode and consume it from the local working
+directory.
+
+Please, first of all, clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/tutorial/developing_packages/editable_packages
+
+There are 2 folders inside this project:
+
+..  code-block:: text
+
+    .
+    ├── hello
+    │   ├── CMakeLists.txt
+    │   ├── conanfile.py
+    │   └── src
+    │       └── hello.cpp
+    └── say
+        ├── CMakeLists.txt
+        ├── conanfile.py
+        ├── include
+        │   └── say.h
+        └── src
+            └── say.cpp
+
+
+- A "say" folder containing a fully fledge package, with its ``conanfile.py``, its source
+  code.
+- A "hello" folder containing a simple consumer project with a ``conanfile.py`` and its
+  source code, which depends on the ``say/1.0`` requirement.
+
+We will put ``say/1.0`` in editable mode and show how the ``hello`` consumer can find ``say/1.0``
+headers and binaries in its local working directory.
+
+Put say/1.0 package in editable mode
+------------------------------------
+
+To avoid creating the package ``say/1.0`` in the cache for every change, we are going to
+put that package in editable mode, creating **a link from the reference in the cache to
+the local working directory**:
+
+.. code-block:: bash
+
+    $ conan editable add say say/1.0
+    $ conan editable list
+    say/1.0
+        Path: /Users/.../examples2/tutorial/developing_packages/editable_packages/say/conanfile.py
+
+
+From now on, every usage of ``say/1.0`` by any other Conan package or project will be
+redirected to the
+``/Users/.../examples2/tutorial/developing_packages/editable_packages/say/conanfile.py``
+user folder instead of using the package from the Conan cache.
+
+Note that the key of editable packages is a correct definition of the ``layout()`` of the
+package. Read the :ref:`package layout() section <reference_conanfile_methods_layout>` to learn more
+about this method. 
+
+In this example, the ``say`` ``conanfile.py`` recipe is using the predefined
+``cmake_layout()`` which defines the typical CMake project layout that can be different
+depending on the platform and generator used.
+
+Now that the ``say/1.0`` package is in editable mode, lets build it locally:
+
+.. include:: ../cmake_presets_note.rst
+
+.. code-block:: bash
+
+    $ cd say
+
+    # Windows: we will build 2 configurations to show multi-config
+    $ conan install . -s build_type=Release
+    $ conan install . -s build_type=Debug
+    $ cmake --preset default
+    $ cmake --build --preset release
```

Comment:
rename presets to conan-

Reply:
let's do the renaming after the PR is merged with a PR to develop2, because this is going to be live before beta10 is released with the change, part the docs for that feature should be updating all the presets used in the 2.0 docs